ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/invoker/ShellSnippetsInvoker.java

```diff
@@ -221,6 +229,7 @@ protected Project getProject(String source, boolean isOffline) throws InvokerExc
      * @throws InvokerException If compilation failed.
      */
     protected PackageCompilation compile(Project project) throws InvokerException {
+        boolean isContainsErrors = false;
```

Comment:
make the variable name be `containErrors`

Reply:
Updated in 9f270db8b7259fb63e77545e8454623db4d6c3a7